### DISCUSSION TITLE
Error #1: scope.sandbox.at(...).property.match is not a function

### DISCUSSION THREAD

**Discussion by corealan** [stoplightio/spectral#2357] [Q&A]
Hello everyone!
I'm trying to run spectral from command line. And use following command:
spectral lint C:\TEST\test.yaml --ruleset C:\TEST\rules.yaml --format junit --output C:/TEST/Lint_report.xml --verbose
But i always got that error:
![image](https://user-images.githubusercontent.com/34137905/205278644-2de29ad3-5f26-467a-967a-d050ac6397f9.png)

Are there anyone faced same problem?

**Comment by rpiechatzek** [comment]
Hi,
I am facing the same problem.
Examples:
Spectral v6.6.0

Swagger:
`swagger: "2.0"
paths:
  /users:
    post:
      summary: Creates a new user.
      parameters:
        - in: body
          name: user
          description: The user to create.
          schema:
            type: object
	    required:
              - userName
            properties:
              userName:
                type: string`

OpenAPI3:
`openapi: "3.0.0"
paths:
  /users:
    get:
      summary: "Lists or finds user objects"
      parameters:
        - name: "users.id"
          schema:
            type: "string"
          required: false
          in: "query"
          description: "usersID. Comma separated list of blabla"`

For both API Spec the CLI version of Spectral fails with
`Error running Spectral!
Use --verbose flag to print the error stack.
Error #1: scope.sandbox.at(...).property.match is not a function`

Without the schema-parameter Spectral CLI works fine ; of course it finds a lot of errors to report because of the simple API Spec ;-)

**Reply by P0lip** [reply]
Could you share the ruleset?
There's probably a number in there, so doing `@property.match` will indeed fail in such a case.
Try doing `@property.toString().match`
